src/components/editMeasure/populationCriteria/groups/QDM/QDMMeasureGroups.tsx

```diff
@@ -526,6 +524,7 @@ const MeasureGroups = (props: MeasureGroupProps) => {
           });
         });
     } else {
+      group.populations[0].id = uuidv4();
```

Comment:
Assigning id to the first population, what does this do?

Reply:
@adongare This is when adding a group, id is required. Thanks Ashok!

Reply:
ignore my lack of QDM understanding. Haven’t reviewed earlier PRs. is it not required for other populations?

Reply:
@adongare No you are right it should be for all populations, I just changed it, thanks so much Ashok!